src/main/kotlin/com/cognifide/gradle/aem/common/instance/AbstractInstance.kt

```diff
@@ -26,8 +29,8 @@ abstract class AbstractInstance(
     override val zoneId: ZoneId
         get() = property("user.timezone")?.let { ZoneId.of(it) } ?: ZoneId.systemDefault()
 
-    override val sync: InstanceSync
-        get() = InstanceSync(aem, this)
+    override val version: String
+        get() = sync.status.productVersion
```

Comment:
this value will be "<unknown>" only for moment when HTTP service is down when instance is getting up